rust/repositories.bzl

```diff
@@ -465,51 +465,56 @@ def _load_rustfmt(ctx):
         target_triple = target_triple,
         tool_name = "rustfmt",
         tool_subdirectories = ["rustfmt-preview"],
-        version = ctx.attr.rustfmt_version,
+        version = rustfmt_version,
     )
 
     return BUILD_for_rustfmt(target_triple)
 
-def _load_rust_compiler(ctx):
+def _load_rust_compiler(ctx, version, iso_date):
     """Loads a rust compiler and yields corresponding BUILD for it
 
     Args:
         ctx: A repository_ctx.
+        version: The rust version
```

Comment:
Nit: please append `for example` with relevant example to version and iso_date here and in other docs as well